.github/workflows/core.yml

```diff
@@ -191,6 +191,8 @@ jobs:
               "$PETSC_DIR"/"$PETSC_ARCH"/externalpackages/git.slepc/src/binding/slepc4py
             pip install --no-deps ngsPETSc netgen-mesher netgen-occt
 
+	    pip install https://github.com/firedrakeproject/Irksome.git
```

Comment:
Actually a better place to put this is in the `ci` optional dependency group in `pyproject.toml` ([here](https://github.com/firedrakeproject/firedrake/blob/master/pyproject.toml#L106)). You want
```
  "IRKsome @ git+https://github.com/firedrakeproject/Irksome.git",
```